### PR TITLE
Update submit-button-modal.php for ETP-119

### PR DESCRIPTION
**Pull request by jentheo**
Use function to get ticket label

### CODE REVIEW

File: src/views/blocks/tickets/submit-button-modal.php [hunk 1]
```diff
@@ -20,8 +20,11 @@
  */
 
 /* translators: %s is the event or post title the tickets are attached to. */
-$title       = sprintf( _x( '%s Tickets', 'Modal title. %s: event name', 'event-tickets' ), get_the_title() );
-$button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
+/* translators: %s: Tickets label */
+$title       = sprintf( _x( '%s %s', 'Modal title. %s: event name', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );
```
Comment (tr1b0t) [inline src/views/blocks/tickets/submit-button-modal.php:24]: * :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title
* :warning: Equals sign not aligned correctly; expected 1 space but found 7 spaces
* :no_entry_sign: Multiple placeholders should be ordered. Expected '%1$s, %2$s', but got %s, %s.

<sub>hash: ceabb0a6e40dbd7d0899558310fd08b9</sub>

File: src/views/blocks/tickets/submit-button-modal.php [hunk 2]
```diff
@@ -20,8 +20,11 @@
...
+
+/* translators: %s: Tickets label */
+$button_text = sprintf( _x( 'Get %s', 'Get selected tickets.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) );
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/submit-button-modal.php:27]:
```suggestion
/* translators: %1$s: Event name, %2$s: Tickets label */
$title = sprintf( _x( '%1$s %2$s', 'Tickets modal title.', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );

/* translators: %s: Tickets label */
$button_text = sprintf( _x( 'Get %s', 'Get selected tickets.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) );
```

File: src/views/blocks/tickets/submit-button-modal.php [hunk 3]
```diff
@@ -20,8 +20,11 @@
  */
 
 /* translators: %s is the event or post title the tickets are attached to. */
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/submit-button-modal.php:22]:
```suggestion
```

File: src/views/blocks/tickets/submit-button-modal.php [hunk 4]
```diff
@@ -19,9 +19,11 @@
...
  */
 
-/* translators: %s is the event or post title the tickets are attached to. */
-$title       = sprintf( _x( '%s Tickets', 'Modal title. %s: event name', 'event-tickets' ), get_the_title() );
-$button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
+/* translators: %1$s: Event name, %2$s: Tickets label */
+$title = sprintf( _x( '%1$s %2$s', 'Tickets modal title.', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );
+
+/* translators: %s: Tickets label */
+$button_text = sprintf( _x( 'Get %s', 'Get selected tickets.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) );
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/submit-button-modal.php:26]:
```suggestion
$button_text = sprintf( _x( 'Get %s', 'Get Tickets button.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) );
```

File: src/views/blocks/tickets/submit-button-modal.php [hunk 5]
```diff
@@ -14,14 +14,17 @@
...
- * @version 4.11.3
+ * @version TBD
  *
  */
 
-/* translators: %s is the event or post title the tickets are attached to. */
-$title       = sprintf( _x( '%s Tickets', 'Modal title. %s: event name', 'event-tickets' ), get_the_title() );
-$button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
+/* translators: %1$s: Event name, %2$s: Tickets label */
+$title = sprintf( _x( '%1$s %2$s', 'Tickets modal title.', 'event-tickets' ), get_the_title(), tribe_get_ticket_label_plural( 'event-tickets' ) );
```
Comment (tr1b0t) [inline src/views/blocks/tickets/submit-button-modal.php:24]: :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: b355865c7fe6d23fdca56801a2f8596e</sub>

**Pull request merged**